### PR TITLE
Invoke player actions with keyboard

### LINKED ISSUE #4: Keyboard controls
Add keyboard controls for
- Pause
- play
- Next
- Previous

### PR DESCRIPTION
**Pull request by Spring3**
## Purpose
* Invoke Play/Pause/Next/Previous actions via keyboard. (resolves #4)

## What
* Space now triggers play/pause, arrows left and right trigger the action to play the previous/next song respectively

## How to Test
* `npm start`, go to http://localhost:3000, add some music to the playlist and then press one of the mentioned buttons

## What to Check
Verify that the following are valid
* Space button can play/pause the music
* Arrow left button plays the previous song on the list
* Arrow right button plays the next song on the list
* The event handler persists when you navigate to the next view
* The event handler is released when the component unmounts

### CODE REVIEW

**Comment by Spring3** [comment]
@ashinzekene I think you mixed the PRs. eslintrc.js was changed [here](https://github.com/ashinzekene/react-music-player/pull/9)

File: src/utils/keyboardEvents.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const ACTION_KEYS = [
+  'ARROWLEFT',
+  'ARROWRIGHT',
+  'SPACE',
+];
```
Comment (ashinzekene) [inline src/utils/keyboardEvents.jsx]: Can you add for arrow up and arrow down to increase and decrease the volume respectively?

Comment (ashinzekene) [reply src/utils/keyboardEvents.jsx]: Just saw that that was not integrated

**Comment by ashinzekene** [comment]
Thanks for contributing

**Pull request merged**